third-party/dnd-wiki/classes/class-weapon-master.xml

```diff
@@ -0,0 +1,454 @@
+			<grant type="Class Feature" id="ID_WIKI_CLASS_FEATURE_WEAPON_MASTER_IMPOSING_FIGHTER" level="14" />
+		</rules>
+		<multiclass id="ID_WIKI_MULTICLASS_WEAPON_MASTER">
+			<prerequisite>Strength 15 or Dexterity 15, and Constitution 13</prerequisite>
+			<requirements>([str:15]||[dex:15]),[con:13]</requirements>
+			<setters>
+				<set name="multiclass proficiencies">Light armor, medium armor, simple weapons, and 1 martial weapon</set>
+			</setters>
+			<rules>
+				<grant type="Grants" id="ID_INTERNAL_GRANT_MULTICLASS" />
+				<grant type="Proficiency" name="ID_PROFICIENCY_ARMOR_PROFICIENCY_LIGHT_ARMOR" />
+				<grant type="Proficiency" name="ID_PROFICIENCY_ARMOR_PROFICIENCY_MEDIUM_ARMOR" />
+				<grant type="Proficiency" name="ID_PROFICIENCY_WEAPON_PROFICIENCY_SIMPLE_WEAPONS" />
+				<select type="Proficiency" name="Martial Weapon Proficiency (Weapon Master)" supports="ID_INTERNAL_WEAPON_CATEGORY_MARTIAL_MELEE||ID_INTERNAL_WEAPON_CATEGORY_MARTIAL_RANGED" number="1"/>
+			</rules>
+		</multiclass>
+	</element>
+	<element name="Favored Weapon" type="Class Feature" source="D&amp;D Wiki" id="ID_WIKI_CLASS_FEATURE_WEAPON_MASTER_FAVORED_WEAPON">
+		<description>
+			<p>For a Weapon master, your weapons are what makes you who you are. Through your life you have adapted to fight with all kinds of weapons but have specific type of weapon as your signature weapons.</p>
+			<p class="indent">You get +1 bonus to attack and damage rolls when you use your favored weapon. Choose one simple or martial weapon type, and another when you reach 11th level.</p>
+			<p class="indent">When you reach 20th level, you have advantage on attacks with your favored weapon.</p>
+		</description>
+		<sheet>
+			<description>You get +1 bonus to attack and damage rolls when you use your favored weapon.</description>
+			<description level="11">You get +1 bonus to attack and damage rolls with two types of weapons as your favored weapons.</description>
+			<description level="20">You get +1 bonus to damage and attack rolls with your favored weapons, you also have advantage on attacks with them.</description>
+		</sheet>
+	</element>
+	<element name="Keen Eye" type="Class Feature" source="D&amp;D Wiki" id="ID_WIKI_CLASS_FEATURE_WEAPON_MASTER_KEEN_EYE">
+		<description>
+			<p>At 2nd level, you gain advantage on Investigation checks related to weapons, armor, other related metal pieces.</p>
+		</description>
+		<sheet>
+			<description>You gain advantage on Investigation checks related to weapons, armor, other related metal pieces.</description>
+		</sheet>
+	</element>
+	<element name="Fighting Style" type="Class Feature" source="D&amp;D Wiki" id="ID_WIKI_CLASS_FEATURE_WEAPON_MASTER_FIGHTING_STYLE">
+		<description>
+			<p>At 2nd level, you adopt a style of fighting as your specialty. Choose one of the following options. You can't take a Fighting Style option more than once, even if you later get to choose again.</p>
+			<h5>ARCHERY</h5>
```

Comment:
missing defense/protection in the description, can be copied from fighter class

Reply:
Added.